db/seeds.rb

```diff
@@ -10,5 +10,9 @@
 Location.create(body: "Back Bay")
 Location.create(body: "South End")
 Location.create(body: "North End")
-Location.create(body: "Fenway")
-Location.create(body: "Allston")
+fenway = Location.create(body: "Fenway")
```

Comment:
in your `seeds.rb` file you can start to use `create!` so that an error message occurs if the item does not validate.